### PR TITLE
Accelerometer Data Reading

### PR DESCRIPTION
**Pull request by AndrewPetsche21**
Accelerometer data is being displayed and is able to be used

### CODE REVIEW

File: src/acc.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include <Wire.h>
+#include <Adafruit_Sensor.h>
+#include <Adafruit_ADXL345_U.h>
+  
+/*Initialize an instance of Adafruit_ADXL345_Unified with a unique id*/
+
+Adafruit_ADXL345_Unified accel = Adafruit_ADXL345_Unified(12345);
+ 
+void setup() {
```
Comment (doudar) [inline src/acc.cpp]: This should be setupACC

File: src/acc.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ 
+  Serial.begin(115200);
+  Serial.println("");
+  Serial.println("Small example to read from ADXL345 accelerometer");
+  accel.begin();
+ 
+}
+ 
+void loop() {
```
Comment (doudar) [inline src/acc.cpp]: And loopACC

File: src/main.cpp [hunk 3]
```diff
@@ -1,52 +1,53 @@
...
+//   if (!res)
+//   {
+//     Serial.println("Failed to connect"); // print results
+//   }
+//   Serial.println("Connection Successful!");
+// }
+
+// void loop()
+// {
+//   digitalWrite(ledPin, ledState);
```
Comment (doudar) [inline src/main.cpp]: And then call loopACC() here

File: src/main.cpp [hunk 4]
```diff
@@ -1,52 +1,53 @@
...
+//   // WiFi Manager setup
+//   WiFiManager wm;
+//   bool res;
+//   res = wm.autoConnect("ESP32", "password"); // ssid and password for access point
+//   if (!res)
+//   {
+//     Serial.println("Failed to connect"); // print results
+//   }
+//   Serial.println("Connection Successful!");
+// }
```
Comment (doudar) [inline src/main.cpp]: Call setupACC() here

**Review by Nick-VanEssen — APPROVED** [review]

**Pull request merged**